lua/terrortown/lang/en.lua

```diff
@@ -2073,3 +2069,26 @@ L.label_spec_prop_dash = "Dash force multiplier"
 L.label_keyhelper_possession_dash = "prop: dash in view direction"
 L.label_keyhelper_weapon_drop = "drop selected weapon if possible"
 L.label_keyhelper_ammo_drop = "drop ammo from selected weapon out of clip"
+
+-- 2023-12-07
```

Comment:
Maybe @TimGoll can quickly run his script on this one to update the other language files.

Otherwise if you are able to, you can also do it on you own.
Run our language tool https://github.com/TTT-2/ttt2-language_parser on the code.

Reply:
https://github.com/TTT-2/TTT2/pull/1162/commits/d4906183c47318c771579dcd79701f03e4374782

Reply:
NIce to see that you're doing such good work and even use our tools. Thank you!